security.rst

```diff
@@ -1846,7 +1846,53 @@ To enable logging out, activate the  ``logout`` config parameter under your fire
             ;
         };
 
-Next, you need to create a route for this URL (but not a controller):
+``path`` can be either a route name, or a URI.
+
+Since Symfony 6.4, you can leverage the
+:class:`Symfony\\Bundle\\SecurityBundle\\Routing\\LogoutRouteLoader`
+to automatically register a route for every logout URI you configured.
+If your project uses :ref:`Symfony Flex <symfony-flex>`,
+this is already done for you.
+Otherwise, you need to import the ``security.route_loader.logout`` service
+as a routing resource:
```

Comment:
With your PR, configure logout is now super easy. I think docs must reflect this.

What do you think to tell first: if you have flex is enough (I'm not sure but most of people use it right ?). 

Then explain more complex cases ?

Reply:
Hm that's how I wanted it but it seems to have failed 😅 

Flex is mentioned first, then how to manually use the `LogoutRouteLoader`, then how to do without it (if you need to internationalize logout URIs e.g.)

As I'll have some rewording to do with https://github.com/symfony/symfony-docs/pull/19000#discussion_r1359909741 I'll try to make this simpler.